docs/preview-environments/drivers/build-image.mdx

```diff
@@ -0,0 +1,60 @@
+---
+id: build-image
+slug: /preview-environments/drivers/build-image
+title: build-image
+---
+
+:::info
+
+- This driver requires the [`target.app_name`](/preview-environments/porter-yaml-reference#resourcetarget) field to be set to the Porter release name.
+- The [`source.name`](/preview-environments/porter-yaml-reference#resourcesource) field should use the name of the Helm chart that the Porter release is going to use.
+
+:::
+
+This preview environments driver can be used to build a Docker image locally by using some details from the intended Porter release that is being deployed using the built image. The [`config`](/preview-environments/porter-yaml-reference#resourceconfig) section, when using this driver, supports the following properties:
+
+- `build` [`Object`, required] - options to build the image, with the following properties:
+  - `use_pack_cache` [`Boolean`, optional] - whether to use the pack cache when building the image, when using the `pack` method
+  - `method` [`String`, required] - one of `docker`, `pack` ([Paketo Buildpacks](https://paketo.io)) or `registry`
+  - `context` [`String`, optional] - the current build context or working directory to use. Defaults to the current working directory.
+  - `dockerfile` [`String`, required only when `method` is `docker`] - path to the Dockerfile to use, relative to the `context`. Defaults to `./Dockerfile`.
```

Comment:
Can we double-check that this is relative to the context? I thought it was relative to the root directory. Context and dockerfile path should be independent and both relative to the root directory, which is what `docker build` uses by default.